plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -881,8 +890,8 @@ func TestLimitRanger_GetLimitRangesFixed22422(t *testing.T) {
 			value.Namespace = action.GetNamespace()
 			limitRangeList.Items = append(limitRangeList.Items, value)
 		}
-		// he always blocking before sending the signal
-		<-unhold
+		// make the handler slow so concurrent calls exercise the singleflight
+		time.Sleep(time.Second)
```

Comment:
this has to be always higher than the time it takes to create the goroutines and hit the group

Reply:
no, if it is not higher, the ones that missed the singleflight hit the cache, and the test still passes